common.blocks/popup/popup.bemhtml

```diff
@@ -8,5 +8,11 @@ block('popup')(
             directions : ctx.directions,
             zIndexGroupLevel : ctx.zIndexGroupLevel
         };
+    }),
+    attrs()(function() {
+        return {
+            role : 'dialog',
+            'aria-hidden' : !this.mods.visible
```

Comment:
Add `aria-hidden` attribute only if it's value is `true` and remove it if `false`.